package.json

```diff
@@ -233,7 +239,7 @@
                 {
                     "command": "vscode-docker.images.prune",
                     "when": "view == dockerImages",
-                    "group": "navigation@2"
+                    "group": "navigation@3"
```

Comment:
This doesn't need to change does it? It's part of the Images view.

Reply:
That is true, not sure how that happened.